rpa-manager/modules/ROOT/pages/processoperations-financeanalysis-analyze.adoc

```diff
@@ -83,11 +83,11 @@ Enter the new monthly operating costs.
 . Click *OK*.
 . Click *Save*.
 
-=== Forcast monthly process runs
+=== Forcast Monthly Process Runs
 
 Expected number of automatically executed process runs in the current month and in coming months. This number initially equals the number of manually executed process runs per month so far (Frequency per month for manual process execution), but can be adjusted at any time.
 
-Estimate the expected number of process runs again every month on the basis of the pattern to date in order to forecast the break-even time as accurately as possible.
+Estimate the expected number of process runs again every month on the basis of the pattern to date to forecast the break-even time as accurately as possible.
```

Comment:
```suggestion
Estimate the expected number of process runs every month on the ability of the pattern to forecast the break-even time as accurately as possible.
```

Reply:
Yes it does :) This is maybe better:
Based on past experience, re-estimate the expected number of process runs every month to forecast the break-even time as accurately as possible.